spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "spec_helper"
+
+describe SessionsController do
+  describe "GET new" do
+    it "redirects to home_path if logged in" do
+      session[:user_id] = Fabricate(:user).id
+      get :new
+      response.should redirect_to home_path
+    end
+    it "renders the new session template if not logged in" do
+      get :new
+      response.should render_template :new
+    end
+  end
+  describe "POST create" do
+    it "finds a valid user with valid email" do
+      user = Fabricate(:user, password: "password")
+      expect(User.where(email: user.email).count).to eq(1)
```

Comment:
You are testing `User.where`, not your `post` route. You should be creating the user with the post route, not a fabricator.